lib/vmdb/loggers.rb

```diff
@@ -26,6 +26,7 @@ def self.apply_config(config)
       apply_config_value(config, $kube_log,          :level_kube)
       apply_config_value(config, $mw_log,            :level_mw)
       apply_config_value(config, $datawarehouse_log, :level_datawarehouse)
+      apply_config_value(config, $monitoring_log,    :level_monitoring)
```

Comment:
I went with container_monitoring for the log and cn_monitoring_log for the logger, so it won't be super long.